code/modules/projectiles/projectile/beams.dm

```diff
@@ -50,7 +50,7 @@
 /obj/item/projectile/beam/scatter
 	name = "laser pellet"
 	icon_state = "scatterlaser"
-	damage = 5
+	damage = 12.5
```

Comment:
This projectile is also used by the scatter laser gun, which is normally not available except from away missions and summon guns events. Is this intentional?